components/06-examples/header.hbs

```diff
@@ -1,15 +1,12 @@
 <header>
   <div class="dso-logo-bar">
     {{> '@logo' }}
-    <div class="dso-tools-bar">
-      {{render '@searchbar' searchbarContext merge=true }}
+    <div class="dso-tools-bar"> 
```

Comment:
Deze change is niet nodig

Reply:
Daar ben ik het mee oneens. Deze change is stiekem best wel belangrijk omdat een algemene zoekbalk bij ons best veel problemen geeft. 

Waarin zou een gebruiker namelijk in het dso gaan zoeken? Werkzaamheden  in de check Werkzaamheden in Aanvragen of Pagina's zoals cookies. 

Er zijn veel discussies geweest over de zoekfunctionaliteit in de header en wanneer dat zou kunnen. Maar we komen er voorlopig op uit dat dit niet zal gaan gebeuren. 

Daarom wil ik ook geen search bar in de header hebben staan.

Reply:
Ik had het over de spatie aan het einde op regel 4:
```
    <div class="dso-tools-bar"> 
                     SPATIE ---^
```

Reply:
Oh hahah helemaal verkeerd geinterpreteerd. Fix ik even.